website/docs/middleware/logger.md

```diff
@@ -11,6 +11,13 @@ Echo has 2 different logger middlewares:
 - Older string template based logger [`Logger`](https://github.com/labstack/echo/blob/master/middleware/logger.go) - easy to start with but has limited capabilities
 - Newer customizable function based logger [`RequestLogger`](https://github.com/labstack/echo/blob/master/middleware/request_logger.go) - allows developer fully to customize what is logged and how it is logged. Suitable for usage with 3rd party logger libraries.
 
+## Dependencies
```

Comment:
I do not think this block is necessary.  You do not need to import these when using `e.Use(middleware.Logger())` or `middleware.LoggerWithConfig(...)`